src/main/scala/org/deepdive/extraction/ExtractorRunner.scala

```diff
@@ -449,12 +449,20 @@ class ExtractorRunner(dataStore: JdbcDataStore, dbSettings: DbSettings) extends
     log.info(s"File dumped to ${actualDumpedPath}")
     val splitPrefix = s"${actualDumpedPath}-"
     val linesPerSplit = task.extractor.inputBatchSize
+    val maxParallel = task.extractor.parallelism
+    val splitNum = maxParallel * 10
```

Comment:
Why do we need `* 10` here?  If I understood correctly, your fix is trying to avoid the startup overhead of UDFs, right?  Then why do we still pay 9 more times if we can just start once and handle the chunk?